README.md

```diff
@@ -71,12 +77,12 @@ You can set `DOCKER_ARCH`, `REGISTRY` and `VERSION` in the commandline to build
 ```
 make image DOCKER_ARCH=amd64 REGISTRY=yunikorn VERSION=latest
 ```
-This command will build an amd64 binary executable with version `latest` and the docker image tag is `yunikorn/yunikorn:scheduler-latest`. If not specified, `DOCKER_ARCH` defaults to the build host's architecture.
+This command will build an amd64 binary executable with version `latest` and the docker image tag is `yunikorn/yunikorn:scheduler-amd64-latest`. If not specified, `DOCKER_ARCH` defaults to the build host's architecture.
```

Comment:
Can we add some language reflecting that this is a architecture-specific tag? It will be different for arm, etc.

Reply:
Done.